packages/core/src/internal-components/panel/panel.element.scss

```diff
@@ -0,0 +1,33 @@
+@import './../../styles/tokens/generated/index';
+
+:host {
+  --width: 100%;
+  --height: 100%;
+  --color: initial;
+  --overflow: visible; // default html behaviour
+  --overflow-x: visible; // default html behaviour
+  --overflow-y: visible; // default html behaviour
+  --background: #{$cds-alias-object-container-background};
+  --border-color: initial;
+  --border-width: #{$cds-alias-object-border-width-100};
+  --border-radius: #{$cds-alias-object-border-radius-100};
```

Comment:
You could simplify style API by combining the border color and width properties as one, that will also allow border type to be overridden.

```css
--border:  #{$cds-alias-object-border-width-100} solid initial;
```

Reply:
That one was something I thought about and then forgot to raised it as a "how to" question in the channel and eventually forgot about it. Thank you! Awesome fix!